explorer/app/transformers/hca.ts

```diff
@@ -37,9 +35,9 @@ export const getProjectId = (value: ProjectResponse) =>
  * Transforms a response to the file endpoint into a viewModel, that will be used by
  * @see DetailContainer
  * @param value Api's response type
- * @returns @see DetailViewModel
+ * @returns @see DetailModel
  */
-export const fileDetailToView = (value: FileResponse): DetailViewModel => ({
+export const fileDetailToView = (value: FileResponse): DetailModel => ({
```

Comment:
You can remove `fileDetailToView` and `sampleDetailToView` as we don't have detail pages for these at the moment.